admin_interface/models.py

```diff
@@ -156,6 +156,35 @@ class Theme(models.Model):
         verbose_name=_("display"),
     )
 
+    css_body_foreground_color = ColorField(
+        blank=True,
+        default="#333",
+        help_text=_("Default: #333 (required fields, error messages)"),
+        max_length=10,
+        verbose_name=_("foreground color"),
+    )
+    css_body_background_color = ColorField(
+        blank=True,
+        default="#fff",
+        help_text="#fff",
+        max_length=10,
+        verbose_name=_("background color"),
+    )
+    css_body_quiet_color = ColorField(
+        blank=True,
+        default="#666",
+        help_text=_("Default: #666 (optional fields, help text, field content)"),
+        max_length=10,
+        verbose_name=_("quiet color"),
+    )
+    css_body_loud_color = ColorField(
+        blank=True,
+        default="#000",
+        help_text="#000",
```

Comment:
Could you improve this `help_text` too please?

Reply:
I couldn't find where loud color was used, 

> I couldn't find places where loud color was used, but it felt awkward to add the option for the quiet color without the loud color.